ImportBlocks/BlockImporter.cs

```diff
@@ -0,0 +1,125 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Akka.Actor;
+using Neo.IO;
+using Neo.Ledger;
+using Neo.Network.P2P.Payloads;
+
+namespace Neo.Plugins
+{
+    public class BlockImporter : UntypedActor
+    {
+        public class StartImport { public IActorRef BlockchainActorRef; public Action OnComplete; }
+        private IActorRef _blockchainActorRef;
+        private const int BlocksPerBatch = 10;
+        private bool isImporting;
+        private IEnumerator<Block> blocksBeingImported;
```

Comment:
We need to call `blocksBeingImported.Dispose()` after import.

Reply:
Nice catch. Fixed.